week2/materials/homework/assignments.md

```diff
@@ -4,38 +4,48 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/rajani-week2-homework?region=us-east-1&tab=objects`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `AWS Certificate Manager` (https://aws.amazon.com/certificate-manager/)
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: `1172 USD (for first 50 TB / Month costs 23usd and after 1TB cost 22usd)`
```

Comment:
Please add come calculations. The values are correct :)

Reply:
I added the calculations...Thank You @paulinadubas